README.md

```diff
@@ -59,6 +59,7 @@ Service Name | Package name
 [IAM Access Groups](https://cloud.ibm.com/apidocs/iam-access-groups) | iamaccessgroupsv2
 [IAM Identity Service](https://cloud.ibm.com/apidocs/iam-identity-token-api) | iamidentityv1
 [IAM Policy Management](https://cloud.ibm.com/apidocs/iam-policy-management) | iampolicymanagementv1
+[IBM Cloud Shell](https://cloud.ibm.com/apidocs/cloudshell) | ibmcloudshellv1
```

Comment:
I'm sort of wondering if the name of the service should just be "Cloud Shell" to match the label on the tile within the platform-services category here (https://test.cloud.ibm.com/docs?tab=api-docs&category=platform_services).
If that is the case, then the x-alternate-name extension could be added to the API definition, like this:
```
 "info": {
    "description": "API docs for IBM Cloud Shell repository",
    "title": "IBM Cloud Shell",
    "x-alternate-name": "Cloud Shell",
    ...
``` 
It's just that within the SDKs, we typically don't include "IBM" in the service names because that is understood (more or less).